harpy/_misc.py

```
@@ -138,4 +139,37 @@ def safe_read(file_path: str):
+    with open(os.path.join(workdir, 'config.yaml'), "w", encoding="utf-8") as sm_config:
+        yaml.dump(profile, sm_config, sort_keys=False, width=float('inf'))
+
+def write_workflow_config(configs, outdir):
+    """
+    Writes a workflow.yaml file to workdir to use with --configfile. Creates outdir/workflow if it doesnt exist. Configs
+    are expected to be a dict
+    """
+    workdir = f"{outdir}/workflow"
+    if not os.path.exists(workdir):
+        os.makedirs(workdir, exist_ok=True)
+    with open(os.path.join(workdir, f'config.harpy.yaml'), "w", encoding="utf-8") as config:
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)